account_asset_management/views/account_asset.xml

```diff
@@ -334,28 +322,30 @@
                     string="Draft"
                     name="draft"
                     domain="[('state', '=', 'draft')]"
-                    help="Draft Assets"
                 />
                 <filter
                     string="Running"
                     name="running"
                     domain="[('state', '=', 'open')]"
-                    help="Assets in Running State"
```

Comment:
Why removing this? Isn't working?

Reply:
It brings 0 additionnal information. It just gives more work for translators for nothing...

Reply:
OK then